app/src/androidTest/java/testapp/NavigationViewTest.java

```diff
@@ -53,4 +59,21 @@ public void onNavigationStart_mapboxNavigationIsNotNull() {
 
     assertNotNull(mapboxNavigation);
   }
+
+  @Test
+  public void onNavigationStart_mapboxSendFeedback() {
```

Comment:
look like there might be room for building abstraction on top of the view?

Reply:
Can you please be a little more specific?

Reply:
i'm referring to page object pattern or more androidy https://jakewharton.com/testing-robots/

Reply:
This is something new for me, but I like it. 👍